arionBackend/tests/api/test_hierarchy.py

```diff
@@ -0,0 +1,51 @@
+"""
+This module holds the test cases for the different views.
+"""
+
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+
+class HierarchyTestCase(APITestCase):
+    """
+    This class contains the tests for the Hierarchy API.
+    """
+
+    @classmethod
+    def setUpTestData(cls):
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename method "setUpTestData" to match the regular expression ^[a-z_][a-z0-9_]{2,}$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.tommartensen.de/coding_rules#rule_key=python%3AS100)